build.gradle

```diff
@@ -172,6 +172,18 @@ dependencies {
     testFixturesImplementation "org.junit.jupiter:junit-jupiter:$junitVersion"
 }
 
+// substitute commons-compress:1.24.0 from testcontainers:1.19.8 to version 1.26.2
```

Comment:
Just a small note: this comes from **test** dependency. why do we need this substitution for all configurations? They are not present in the production distribution.
Also, we must remember now to remove this block once the **testcontainers** dependency is updated and no longer contains this vulnerability. The best way would be to exclude the test configurations from scanning - we don't get alerts on dependencies that we don't care about and we don't need to do things like this